psutil/_psosx.py

```diff
@@ -212,6 +213,84 @@ def disk_partitions(all=False):
 # =====================================================================
 
 
+def get_smc_temperatures(smc_keys, iter_num, iteratable):
+    """ Receives a dictionary of SMC keys, and a number possible keys
+    If keys can be iterated, replaces the expected %d in the key.
+    Returns a list of all smc_get_temperature results higher than 0
+    """
```

Comment:
Please add a reference/link which shows the possible key values.
Something like this:
```python
def get_smc_temperatures(smc_keys, iter_num, iteratable):
    """ Receives a dictionary of SMC keys, and a number of possible keys.
    If keys can be iterated, replaces the expected %d in the key.
    Returns a list of all smc_get_temperature results higher than 0.
    References:
 
    https://github.com/Chris911/iStats/blob/
        09b159f85a9481b59f347a37259f6d272f65cc05/lib/iStats/smc.rb

    http://web.archive.org/web/20140714090133/http://www.parhelia.ch:80/
        blog/statics/k3_keys.html
    """